### PR TITLE
Saltern Additions

### PR DESCRIPTION
**Pull request by SweptWasTaken**
-Maps in Grinder for Kitchen
-Other tiny stuff

### CODE REVIEW

File: Resources/Maps/saltern.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 meta:
   format: 2
-  name: Saltern
-  author: Buncha sickass mfers
+  name: DemoStation
```
Comment (Zumorica) [inline Resources/Maps/saltern.yml:3]:
```suggestion
  name: Saltern
```

File: Resources/Maps/saltern.yml [hunk 2]
```diff
@@ -47599,4 +47460,722 @@ entities:
...
+        type: Robust.Server.GameObjects.Components.Container.Container
+      body_bag_label:
+        type: Content.Server.GameObjects.ContainerSlot
+    type: ContainerContainer
+- uid: 4266
+  type: FoodMemoryLeek
+  components:
+  - parent: 855
+    pos: -10.516438,0.52631223
+    type: Transform
```
Comment (metalgearsloth) [inline Resources/Maps/saltern.yml:48078]: IMO this shouldn't be in bar, especially when it can mess with reagents.

**Review by metalgearsloth — APPROVED** [review]
I only had a quick look in game FWIW

File: Resources/Maps/saltern.yml (hunk 2 above)
Comment (Zumorica) [reply Resources/Maps/saltern.yml:48078]: Agreed.

**Comment by Erisfiregamer1** [comment]
Might want to merge the latest commit. You have one that has a failing Windows build.

**Comment by SweptWasTaken** [comment]
Removed leak, should be good for merge.

**Pull request merged**